### DISCUSSION TITLE
Good first issues in the docs repo

### DISCUSSION THREAD

**Discussion by CsatariGergely** [dita-ot/dita-ot#4262] [Q&A] [answered]
Hi, we would like to help the community to have better documentation and looking for good first issues on the docs repo. The current one are all looking very old and difficult. Can someone suggest some easy changes what we could address as a start?
Thanks.

**Comment by stefan-jung** [comment]
I would say the docs are almost bug free and they are really good. The list of [issues](https://github.com/dita-ot/docs/issues) is almost empty. @infotexture and others do a perfect job. Not sure if a big reworking is needed (this has happened already). Probably the challenge is to keep the docs as good as they are and I am sure that the biggest effort is to write the release notes.

**Reply by CsatariGergely** [reply]
Thanks for the answer. This is good and bad at the same time :) @infotexture if you have any are in mind what we could work on please notify me.

**Comment by infotexture** [comment, marked as answer]
@CsatariGergely 🙏 Thanks for your interest in the DITA-OT docs.

We haven’t maintained a `good first issue` or `help wanted` label in the docs backlog lately, as the nature of a tool like DITA-OT means there’s a certain learning curve for the documentation process.

While some projects mark certain issues as _“beginner friendly”_, _“easy bug fix”_, or _“low-hanging-fruit”_ and leave them open to encourage contributions, we generally prefer to fix things as soon as possible to ensure that the next version of the software ships with any suggestions we receive.

That said, there are always suggestions that don’t get implemented, as the original reporter may not have had the time to propose changes, and others may not have known how. To keep our backlog manageable, issues that aren’t updated get marked as `stale` after a while, and eventually closed if no further activity occurs. (For examples, set the [`is:issue is:closed label:stale`](https://github.com/dita-ot/docs/issues?q=is%3Aissue+is%3Aclosed+label%3Astale) issue filters.) Anything there is fair game, if something sparks your interest.

Apart from that, we have a few tips in the [Documentation](https://www.dita-ot.org/contributing#documentation) section under _Contributing to DITA Open Toolkit_ on the project website.

We’re open to suggestions, so if there’s anything your team has struggled with, [file a docs issue](https://github.com/dita-ot/docs/issues/new) (or [send a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests)) and we’ll be happy to discuss how to make things easier to understand.

**Reply by CsatariGergely** [reply]
Great, thanks. Will look into the staled issues. 
We are using DITA-OT and have some very limited tech writer capacity to help the project.
